docs/credentials/EcommerceInvoiceRegistrationCredential.json

```diff
@@ -0,0 +1,37 @@
+{
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1",
+    "https://w3id.org/traceability/v1"
+  ],
+  "id": "https://example.com/123",
+  "type": [
+    "VerifiableCredential"
+  ],
+  "issuer": {
+    "id": "did:key:z6MkoTHsgNNrby8JzCNQ1iRLyW5QQ6R8Xuu6AA8igGrMVPUM"
+  },
+  "issuanceDate": "2020-03-10T04:24:12.164Z",
+  "expirationDate": "2029-03-10T04:24:12.164Z",
+  "credentialSubject": {
+    "id": "did:web:www.amazon.com",
```

Comment:
hmm, this might get us in trouble, prefer example.com

Reply:
updated